core/templates/dev/head/css/oppia.css

```diff
@@ -704,6 +704,22 @@ textarea {
   }
 }
 
+.navbar-nav.oppia-navbar-nav > li > a.oppia-navbar-editor-navigation-toggle{
```

Comment:
Please follow the way existing selectors are styled (space before '{', space after ',', multiple selectors on separate lines if they're too long, etc.)